### PR TITLE
fix: update text in handoff modals

### LINKED ISSUE #6566: Create Copy for Share Resource Request Instructions
## Is your feature request related to a problem? Please describe.

The Copy on the Share Resource Request Instructions needs to be determined and created

## Additional context

image

### LINKED ISSUE #6567: Create Copy for Share Resource Request Modal - LUIS / QnA Dev Setup
## Is your feature request related to a problem? Please describe.

The Copy on the following modals needs to be determined and created

- [ ] LUIS
- [ ] QnA
- [ ] Speech

## Additional context

image

### LINKED ISSUE #6867: Resource request handoff is incorrect for QnA resource
## Describe the bug

When requesting a resource request handoff for QnAMaker, the instructions for the Azure admin are for LUIS?

## Version

1.4.0-nightly.233776

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Create basic assistant
2. From rocket ship choose QnA
3. Generate resource request

## Expected behavior

QnA instructions

## Screenshots

image

image

### PR DESCRIPTION
**Pull request by benbrown**
## Description

Adds the handoff instructions for LUIS, QNA, Speech and provision process.

## Task Item

Fixes #6567 
Fixes #6566 
Fixes #6867 

## Screenshots
Screen Shot 2021-04-13 at 11 38 23 AM

### CODE REVIEW

File: Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx [hunk 1]
```diff
@@ -379,7 +375,7 @@ export const ManageQNA = (props: ManageQNAProps) => {
       } catch (err) {
         setOutcomeDescription(
           formatMessage(
-            'Due to the following error, we were unable to successfully add your selected LUIS keys to your bot project:'
+            'Due to the following error, we were unable to successfully add your selected QNA keys to your bot project:'
```
Comment (beyackle) [inline Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx:378]: `QnA`

File: Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx [hunk 2]
```diff
@@ -392,7 +388,7 @@ export const ManageQNA = (props: ManageQNAProps) => {
       setLoadingQNA(false);
 
       setOutcomeDescription(
-        formatMessage('The following LUIS resource was successfully created and added to your bot project:')
+        formatMessage('The following QNA resource was successfully created and added to your bot project:')
```
Comment (beyackle) [inline Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx:391]: `QnA`

**Review by beyackle — CHANGES_REQUESTED** [review]
Looks good apart from one teeny thing; everywhere else I've seen `QnA`, the n is lowercase.

**Review by GeoffCoxMSFT — APPROVED** [review]

**Review by beyackle — APPROVED** [review]

**Comment by hatpick** [comment]
Shouldn't the back button not be primary (not blue)? @benbrown

**Pull request merged**